root/etc/cont-init.d/20-drupal-files-dirs

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/with-contenv bash
+
+for DIR in $(ls -d ${DOCROOT}/sites/*/files); do
+  s6-echo -- Fixing ownership and permissions of $DIR
+  chown apache:apache $DIR
```

Comment:
So I think most people don't even look at it but I figured out a way to detect if it is an nfs filesystem and swallow the messages rather than have to put a big "this may not be an issue if you are operating under condition XYZ. What do you think?